doc-content/enterprise-only/project-data/delete-branches-proc.adoc

```diff
@@ -7,6 +7,6 @@ You can delete any branch except for the *master* branch. {CENTRAL} does not all
 . Click image:project-data/dots.png[] in the upper-right corner of the screen and select *Delete Branch*.
 +
 .Deleting a branch menu
-image::getting-started/delete-branch-menu.png[Delete a branch menu]
```

Comment:
Could you please also remove the  _delete-branch-menu.png_ file? (assuming it's no longer needed)